plugin/pkg/scheduler/factory/factory.go

```diff
@@ -229,6 +234,11 @@ func (f *ConfigFactory) CreateFromKeys(predicateKeys, priorityKeys sets.String,
 	// Cache this locally.
 	cache.NewReflector(f.createControllerLW(), &api.ReplicationController{}, f.ControllerLister.Store, 0).RunUntil(f.StopEverything)
 
+	// Watch and cache all ReplicaSet objects. Scheduler needs to find all pods
+	// created by the same services or ReplicaSets, so that it can spread them correctly.
```

Comment:
s/services or ReplicaSets/services or ReplicationControllers or ReplicaSets/

also, please make the same fix on L233 and L228